package.json

```diff
@@ -14,5 +14,8 @@
     "eslint": "^6.8.0",
     "mocha": "^7.1.2"
   },
-  "dependencies": {}
+  "dependencies": {
+    "chart.js": "^2.9.4",
+    "moment": "^2.29.1"
```

Comment:
I see moment was brought in as a dependency, but I do not see it being used in implementation code.